public/data/knyadzayo.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Kelvin Nyadzayo",
+  "type": "personal",
+  "bio": "Hi, there I'm a software engineer and technical writter from  Zimbabwe. I love to inspire the community by delivering talks and writing blogs on web development & blockchain  topics. ",
+  "avatar": "https://github.com/Nyadzayo",
+  "links": [
+    {
+      "name": "Connect with me on my Website",
+      "url": "https://kelvinnyadzayo.com",
+      "icon": "website"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Nyadzayo",
+      "icon": "github"
+    },
+
+    {
+      "name": "My YouTube Channel",
+      "url": "https://www.youtube.com/channel/UCVx38C7OXK3QU7HH7YWlpLQ/featured",
+      "icon": "youtube"
+    },
+
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/CokeKyles",
```

Comment:
Make sure you've entered your correct `username` in the Twitter link because it shows `account doesn't exists`! 

**Please update your Twitter link with the correct one, else remove it!!**  🛠

![image](https://user-images.githubusercontent.com/88786642/182283567-2976eb64-b316-422d-ad45-c824c9c2c9af.png)




__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
@kumarsonsoff3 thank you for the feedback i have resolved the changes and updated the commits